docs/windows_developer_guide.md

```diff
@@ -61,14 +61,6 @@ python --version
 pip install numpy==1.23
 ```
 
-## MSYS
```

Comment:
Please also move NPM after proxy setting as this is also required for npm

Reply:
Actually setting proxy via export/setting variables didn't work. We need first to set it with npm:
nvm proxy http://SOME_PROXY
then use other npm commands